TweetMate/src/main/java/com/ybsystem/tweetmate/adapters/pager/UserListPagerAdapter.java

```diff
@@ -34,11 +35,11 @@ public int getCount() {
     public CharSequence getPageTitle(int position) {
         switch (position) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)